xml/System.Collections.Generic/Dictionary`2.xml

```diff
@@ -331,9 +331,10 @@
         <Parameter Name="collection" Type="System.Collections.Generic.IEnumerable&lt;System.Collections.Generic.KeyValuePair&lt;TKey,TValue&gt;&gt;" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1;netcore-3.1;net-5.0" />
       </Parameters>
       <Docs>
-        <param name="collection">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="collection">The <see cref="T:System.Collections.Generic.IEnumerable`1" />  whose elements are copied to the new <see cref="T:System.Collections.Generic.Dictionary`2" />.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Collections.Generic.Dictionary`2" /> class that contains elements copied from the specified <see cref="T:System.Collections.Generic.IEnumerable`1" />.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="System.ArgumentException"><paramref name="collection" /> contains one or more duplicated keys.</exception>
```

Comment:
Hey @layomia did you manually add the exceptions or did you port them with the tool? I ask because the exception crefs are incomplete, and if it's the tool's fault, I should get it fixed. This is how it should look:

```suggestion
        <exception cref="T:System.ArgumentException">
          <paramref name="collection" /> contains one or more duplicated keys.</exception>
```

Reply:
I added them manually in https://github.com/dotnet/dotnet-api-docs/pull/4121/commits/e42c4c83cca53bd5a1bf5a777c2ac09e4a26c2d1, but used the tool to port the in the new commit. It missed the one added in `Queue1.xml` (not sure how), so I applied your suggestion in https://github.com/dotnet/dotnet-api-docs/commit/e0f79cf9d063cbb72e045884b5a8d277cb55e5d9 and rebased.

The command I used was:

`DocsPortingTool -docs D:\repos\dotnet-api-docs\xml -tripleslash D:\repos\dotnet_runtime\artifacts\bin\coreclr\Windows_NT.x64.Release\IL\,D:\repos\dotnet_runtime\artifacts\bin\ -includedassemblies System.Collections,System.Collections.Generic,mscorlib -excludedassemblies System.Collections.Immutable,System.Collections.Specialized,System.Collections.Concurrent -save true -skipExceptions false`